cmd/copyEnumeratorInit.go

```diff
@@ -255,6 +234,20 @@ func (cca *CookedCopyCmdArgs) initEnumerator(jobPartOrder common.CopyJobPartOrde
 			if cName == "" || (srcLevel == ELocationLevel.Service() && dstLevel > ELocationLevel.Service()) {
 				cName, err = containerResolver.ResolveName(object.ContainerName)
 
+				//attempt creating a container. We'll not create it multiple times.
+				err = cca.createDstContainer(cName, cca.Destination, ctx, existingContainers, azcopyLogVerbosity)
+
+				// if JobsAdmin is nil, we're probably in testing mode.
+				// As a result, container creation failures are expected as we don't give the SAS tokens adequate permissions.
+				// check against seenFailedContainers so we don't spam the job log with initialization failed errors
+				if _, ok := seenFailedContainers[cName]; err != nil && jobsAdmin.JobsAdmin != nil && !ok {
```

Comment:
(Unrelated to this PR)

This functionality often isn't used, and more frequently confuses the heck out of customers and support engineers alike. We should just stop outputting errors from this on the default logging level to help avoid the confusion.